python/features/calc.feature

```diff
@@ -0,0 +1,11 @@
+Feature:  Calculator example model with requirements
```

Comment:
This would be a good enhancement, but the intent here was just to automate previous manual testing.  Suggest we write a story to add "sad path" testing to the test automation.

Reply:
Agree with that "meet half way" suggestion, incorporating it in the unit test feature.